modules/sector_page/config/optional/core.entity_form_display.node.sector_page.default.yml

```diff
@@ -9,7 +9,9 @@ dependencies:
     - field.field.node.sector_page.field_image_hero
     - field.field.node.sector_page.field_subtitle
     - field.field.node.sector_page.field_text_placement
+    - field.field.node.sector_page.field_title_caption
     - node.type.sector_page
+    - workflows.workflow.sector_editorial
```

Comment:
can you remove this line - I'm sure this workflow is a part of sector_workflow module, which is installed *after* sector_page. Though if it `composer create-project`'s ok, this might not be an issue